packages/gasket-plugin-config/README.md

```diff
@@ -41,13 +41,13 @@ environments in apps.
 
 ### Environment files
 
-By default, this plugin imports files from your `/config` directory to assemble
+By default, this plugin imports files from your `/config` (or `./src`) directory to assemble
```

Comment:
The goal is to support special directories under src, meaning `/config/src` would be what's desired here. However, because users can adjust the config dir location via existing `gasket.config.js`, this change is not needed.